schema.sql

```diff
@@ -727,6 +734,7 @@ CREATE OR REPLACE ACTION create_credentials_by_dwg(
 
     $dwg_result = idos.dwg_verify_owner(
```

Comment:
We need to send `$dwg_owner_wallet_type` as another argument to `dwg_verify_owner`